tests/chainer_tests/links_tests/loss_tests/test_negative_sampling.py

```diff
@@ -108,68 +96,4 @@ def test_backward_cpu_gpu(self):
         testing.assert_allclose(x.grad, xg.grad, atol=1.e-4)
 
 
-class TestNegativeSamplingIgnoreMask(TestNegativeSampling):
```

Comment:
I don't think we should remove this unit test. Could you let me know the reason?